MomentumDiscordBot/Commands/Moderator/ModeratorModule.cs

```diff
@@ -47,5 +49,20 @@ public async Task GetMembersWithRoleAsync(InteractionContext context, [Option("r
                 await ReplyNewEmbedAsync(context, "That role does not exist in this server", DiscordColor.Orange);
             }
         }
+
+        [SlashCommand("status", "Sets the bots status")]
+        public async Task StatusAsync(InteractionContext context,
+            [Option("status", "status")] string status,
+            [ChoiceProvider(typeof(ActivityTypeChoiceProvider))][Option("type", "ActivityType")] string type = null)
+        {
+            DiscordActivity activity;
+            if (Enum.TryParse(type, out ActivityType activityType))
```

Comment:
This seems like a reasonable place for a ternary i.e.
```c#
activity = Enum.TryParse(type, out ActivityType activityType) ? new DiscordActivity(status, activityType) : new DiscordActivity(status);
```

Reply:
In fact you can then just merge with the above line w/ 
```c#
 var activity = Enum.TryParse(type, out ActivityType activityType) ? new DiscordActivity(status, activityType) : new DiscordActivity(status);
```

Reply:
I was thinking about using something like this but I think it gets hard to read really quick. But with newlines I like it. Hope you don't mind.

Reply:
Oh sure, newlines in this case definitely more readable. I'm too used to JS where I use ternaries absolutely everywhere and get very used to reading them lol